src/status_im2/contexts/onboarding/common/carousel/animation.cljs

```diff
@@ -16,28 +17,48 @@
     (js-obj "duration" (or duration progress-bar-animation-duration)
             "easing"   (:linear reanimated/easings)))))
 
+(defn calculate-remainder
+  [dividend divisor]
+  (if (and (not= 0 dividend) (zero? (mod dividend divisor)))
+    divisor
+    (mod dividend divisor)))
```

Comment:
This would be better if it lived in maybe a `utils.math` namespace

Reply:
@ibrkhalil This is very specific to calculate the reminder which has to be used here only, and by name it could have created confusion when placed in generic utilities. Should we still export it from `utils.math`?